Lib9c/Action/Factory/HackAndSlashFactory.cs

```diff
@@ -0,0 +1,85 @@
+using System;
+using System.Collections.Generic;
+using Libplanet;
+using Nekoyume.Action.Interface;
+
+namespace Nekoyume.Action.Factory
+{
+    public static class HackAndSlashFactory
+    {
+        public static IHackAndSlash CreateByBlockIndex(
+            long blockIndex,
+            Address avatarAddress,
+            int worldId,
+            int stageId,
+            List<Guid> costumeIds = null,
+            List<Guid> equipmentIds = null,
+            List<Guid> consumableIds = null,
+            List<RuneSlotInfo> runeSlotInfos = null,
```

Comment:
How about enabling the `#nullable enable` feature and turning those types as `List<T>?`?

Reply:
> How about enabling the `#nullable enable` feature and turning those types as `List<T>?`?

Is it OK to set those values as just `List<T>`, not `List<Guid>` or `List<RuneSlotInfo>`?
I don't have deep knowledge about this, so I need your help :pray:

Reply:
Ah, I mean `List<Guid>` and `List<RuneSlotInfo>` but with the `?` suffix.

The nullable types like `List<Guid>?` and `List<RuneSlotInfo>?`.